### ISSUE TITLE
Some live stream(480p) in HLS output can't be played in ExoPlayer with v0.2.0

### ISSUE THREAD

**Issue by Romantic-LiXuefeng** [google/shaka-streamer#36]
Using the default config file to generate HLS LIVE output.
`/config_files/input_looped_file_config.yaml`  and  `/config_files/pipeline_live_config.yaml`

With version v0.1.0, the multi-bitrates(720p/480p) outputs can be played in `ExoPlayer`.
but with version  v0.2.0, the 480p  output stream can't be played. The `Exoplayer` mp4 demux the resolution is 1129 * 480, and decoder don't support the resolution.

```
/MediaCodecInfo: NoSupport [sizeAndRate.support, 1129x480x-1.0] [OMX.MTK.VIDEO.DECODER.AVC, video/avc] [kungfupanda, MiBOX3_PRO, Xiaomi, 22]
```

Then I manually reset the resolution to 1130 * 480. The player can work well. 
 And transcode with version v0.1.0, the 480p output stream's resolution is 1130 * 480. so the player can play it.
So I think there is some params add with x264 caused this bug in version  v0.2.0.  I'm not familiar with x264. Hope you could point out the reason.

The `TranscoderNode.log`  with V0.1.0: 
```
 ffmpeg -y -stream_loop -1 -re -thread_queue_size 200 -i Sintel.2010.720p.mkv -stream_loop -1 -re -thread_queue_size 200 -i Sintel.2010.720p.mkv -map 0:v:0 -preset ultrafast -f mpegts -pix_fmt yuv420p -flags +cgop -profile:v high -an -c:v h264 -b:v 3M -keyint_min 96 -g 96 -vf "scale=-2:720" /var/folders/2_/nq2wktrj2kv9_p2z8qb3bv2m0000gn/T/shaka-live-i6h62zf2/59fef6e4-7638-4c38-9078-45ac741e2a62 -map 0:v:0 -preset ultrafast -f mpegts -pix_fmt yuv420p -flags +cgop -profile:v main -an -c:v h264 -b:v 2M -keyint_min 96 -g 96 -vf "scale=-2:480" /var/folders/2_/nq2wktrj2kv9_p2z8qb3bv2m0000gn/T/shaka-live-i6h62zf2/6508a3ee-8eca-48a8-a7e2-e2bff8bca624 -map 1:a:0 -vn -ac 2 -c:a aac -b:a 128k -f mpegts /var/folders/2_/nq2wktrj2kv9_p2z8qb3bv2m0000gn/T/shaka-live-i6h62zf2/d1d91e62-5a61-4a1b-8d6e-4be4aadc6753
```

The `TranscoderNode.log`  with V0.2.0: 
```
ffmpeg -y -stream_loop -1 -re -thread_queue_size 200 -i Sintel.2010.720p.mkv -stream_loop -1 -re -thread_queue_size 200 -i Sintel.2010.720p.mkv -map 0:0 -an -cmp chroma -f mpegts -preset ultrafast -c:v h264 -b:v 3M -bf 0 -pix_fmt yuv420p -flags +cgop -profile:v main -level:v 3.1 -keyint_min 96 -g 96 -vf "scale=-2:720" /var/folders/2_/nq2wktrj2kv9_p2z8qb3bv2m0000gn/T/shaka-live-q1gsnksv/090af60e-360d-45ff-8284-c049c8cb7548 -map 0:0 -an -cmp chroma -f mpegts -preset ultrafast -c:v h264 -b:v 2M -bf 0 -pix_fmt yuv420p -flags +cgop -profile:v main -level:v 3.1 -keyint_min 96 -g 96 -vf "scale=-2:480" /var/folders/2_/nq2wktrj2kv9_p2z8qb3bv2m0000gn/T/shaka-live-q1gsnksv/7155ecd1-ec7c-49c6-be60-c654da3771fa -map 1:1 -vn -ac 2 -f mpegts -c:a aac -b:a 128k /var/folders/2_/nq2wktrj2kv9_p2z8qb3bv2m0000gn/T/shaka-live-q1gsnksv/d788b0de-9974-4433-b2c2-19fff847b3b9
```

**Comment by ojw28** [comment]
ExoPlayer is getting the resolution from the HLS master playlist, where it's incorrectly suggesting the width is 1129 (the true height of the media itself is 1130).

**Comment by Romantic-LiXuefeng** [comment]
> The Exoplayer mp4 demux the resolution is 1129 * 480, and decoder don't support the resolution.


Yes, It's a mistake, @ojw28  is correct.  The mp4 demux extract the resolution is 1130 * 480.  Only the output HLS master playlist suggests incorrectly . 
```
#EXT-X-STREAM-INF:BANDWIDTH=8451286,AVERAGE-BANDWIDTH=2174966,CODECS="avc1.42c01f,mp4a.40.2",RESOLUTION=1129x480,AUDIO="default-audio-group"
stream_0.m3u8
```

**Comment by joeyparrish** [comment]
I can reproduce this issue.

I believe I may be able to work around it by tweaking the FFmpeg command-line around scaling.  If so, it might be appropriate to close the associated Packager bug you opened.

**Comment by joeyparrish** [comment]
This is now fixed in Shaka Streamer through an extra filter in FFmpeg.  The fix will be out in v0.3.0 on PyPI later today.

**Comment by Romantic-LiXuefeng** [comment]
@joeyparrish  @ojw28 Thanks for your help!!
